<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QMainWindow" name="Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>1250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>1180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>1250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_timer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>1165</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background:transparent;
	border: 0px;
	color:gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_create_img">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>1165</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color:black;
    background-color: white;
    border-width: 0px;
    border-radius: 10px;
}

QPushButton:disabled{
    color:black;
    background-color: gray;
    border-width: 0px;
    border-radius: 10px;
}

QPushButton:pressed{
    color:black;
    background-color: gray;
    border-width: 0px;
    border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Screenshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_open_file">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>720</y>
      <width>151</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_open_file{
    color:black;
    background-color: white;
    border-width: 0px;
    border-radius: 10px;
}

QPushButton#button_open_file:pressed{
    color:black;
    background-color: grey;
    border-width: 0px;
    border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_filename">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>720</y>
      <width>461</width>
      <height>42</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#input_filename{
    color:black;
    background-color: white;
    border-width: 0px;
    border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="info_of_object">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>780</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#info_of_object {
	background:transparent;
	border: 0px;
	color:gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>vertexes: 0       	facets: 0</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="object_observer_window">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>940</y>
      <width>701</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="style_panel"/>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>938</y>
      <width>700</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1140</y>
      <width>700</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>820</y>
      <width>700</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line#line_4{
    color:#1C1C1C;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>820</y>
      <width>701</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="modification_panel"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
